sphinx/source/docs/user_guide/geo.rst

```diff
@@ -30,10 +30,11 @@ Bokeh's ``GeoJSONDataSource`` can be used almost seamlessly in place of Bokeh's
     geo_source = GeoJSONDataSource(geojson=geojson)
 
     p = figure()
-    p.circle(alpha=0.9, source=geo_source)
+    p.circle(x='x', y='y', alpha=0.9, source=geo_source)
```

Comment:
Apparently this doesn't work as I expected anymore:
```
In [1]: from bokeh.plotting import *

In [2]: p = figure()

In [3]: g = p.line()

In [4]: map(type, [g.glyph.x, g.glyph.y])
Out[4]: [NoneType, NoneType]
```
This used to give `['x', 'y']`.